docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -40,6 +41,66 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.4.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  March 13, 2025
+
+### New Features and Changes
+* __New CNV Categories__:
+    * The genes table and its associated TSV download in __Mutation Frequency__ have been updated with the new CNV categories. <!--PEAR-2135/2137-->
+    * The Cancer Distribution tool in the __Gene Summary Page__ has been updated to reflect the new CNV categories. <!--PEAR-2136/2291-->
+* __Cohort Builder__:
+    * Search behavior has been enhanced by automatically populating the search bar of selected cards as appropriate. Additionally, search accuracy has been improved by ensuring that additional values matching the search input will be returned in the results. <!--PEAR-2107/2274/2274-->
+    * Child Pugh Classification, Ishak Fibrosis Score, and Weiss Assessment Score have been added as default cards. <!--PEAR-2322-->
+    * The deprecated property, Tumor Code, is no longer a default card. <!--PEAR-2332-->
+* __Clinical TSV and JSON__:
+    * Other Clinical Attribute properties are now available.  <!--PEAR-1984-->
+    * TSV headers have been standardized. <!--PEAR-1984-->
+    * All properties directly associated with a case, such as Primary Site and Disease Type, are now included <!--PEAR-2304-->
```

Comment:
i don't love this wording, but not sure if users would know what the case node is.